net-p2p/retroshare/retroshare-0.6.9999.ebuild

```diff
@@ -126,19 +83,27 @@ pkg_preinst() {
 	local ver
 	for ver in ${REPLACING_VERSIONS}; do
 		if ! version_is_at_least 0.5.9999 ${ver}; then
-			elog "You are upgrading from Retroshare 0.5.* to ${PV}"
-			elog "Version 0.6.* is backward-incompatible with 0.5 branch"
-			elog "and clients with 0.6.* can not connect to clients that have 0.5.*"
-			elog "It's recommended to drop all your configuration and either"
-			elog "generate a new certificate or import existing from a backup"
+			ewarn "You are upgrading from Retroshare 0.5.* to ${PV}"
+			ewarn "Version 0.6.* is backward-incompatible with 0.5 branch"
+			ewarn "and clients with 0.6.* can not connect to clients that have 0.5.*"
+			ewarn "It's recommended to drop all your configuration and either"
+			ewarn "generate a new certificate or import existing from a backup"
 			break
 		fi
-		if version_is_at_least 0.6.0 ${ver}; then
-			elog "Main executable was renamed upstream from RetroShare06 to retroshare"
+		if version_is_at_least 0.6.0 ${ver} && ! version_is_at_least 0.6.4 ${ver}; then
+			elog "Main executable has been renamed upstream from RetroShare06 to retroshare"
 			break
 		fi
 	done
-	gnome2_icon_savelist
+
+	if ! use sqlcipher; then
```

Comment:
That sounds like something you may want to know before building it. So `pkg_pretend`? (remember to check `MERGE_TYPE`)

Reply:
To me it seems the warning should be showed for any value of MERGE_TYPE